build.cmd

```diff
@@ -0,0 +1 @@
+dotnet run --project ./build/build.fsproj -- -t %*
```

Comment:
@xperiandri I guess -t means --target? let's prefer to use longer flag names, shorter ones can be used when using the terminal to type faster, but for scripts we should favour readability

Reply:
fixed